Tribler/Core/Utilities/network_utils.py

```diff
@@ -82,20 +82,21 @@ def _test_port(family, sock_type, port):
     assert sock_type in (socket.SOCK_DGRAM, socket.SOCK_STREAM), "Invalid sock_type value %s" % sock_type
     assert 0 < port <= 65535, "Invalid port value %s" % port
 
-    s = None
     try:
-        s = socket.socket(family, sock_type)
-        if sock_type == socket.SOCK_STREAM:
-            s.setsockopt(socket.SOL_SOCKET, socket.SO_LINGER, struct.pack('ii', 1, 0))
-        s.bind(('', port))
+        # In py2 socket lacks the __exit__ dunder method needed for with statements,
```

Comment:
As we do not support py2 anymore, the comment is unnecessary. Just do the py3 thing.

Reply:
Removed